x/rollup/keeper/msg_server.go

```diff
@@ -18,6 +18,11 @@ import (
 	"github.com/samber/lo"
 )
 
+const (
+	MinGasLimit = params.TxGas
+	MaxGasLimit = params.MaxGasLimit
```

Comment:
We should probably clarify that these are for the tx gas limit and not the block gas limit. Could you rename these to something like `MinTxGasLimit`/`MaxTxGasLimit`?